src/Rule/Compare.php

```diff
@@ -130,12 +131,16 @@ public function getMessage(): string
             '>=' => 'Value must be greater than or equal to "{targetValueOrAttribute}".',
             '<' => 'Value must be less than "{targetValueOrAttribute}".',
             '<=' => 'Value must be less than or equal to "{targetValueOrAttribute}".',
-            default => throw new RuntimeException("Unknown operator: $this->operator."),
         };
     }
 
     public function getOptions(): array
     {
+        $targetValueOrAttribute = $this->targetValue;
```

Comment:
Suggestion from @vjik: revert and add test with both `targetValue` and `targetAttribute` passed.